### PR TITLE
Single Self Guided Tour Screen

### PR DESCRIPTION
**Pull request by alefaissal**
# Single Self Guided Tour Screen
-  Map with markers from Audio
- Style
- Still missing Reviews feature on it.

### CODE REVIEW

File: client/js/screens/SingleSelfGuided/SingleSelfGuided.js [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+import {View, Image, Text, ScrollView, TouchableOpacity} from 'react-native';
+import ButtonDefault from '../../components/ButtonDefault';
+import styles from './styles';
+import SafeAreaView from 'react-native-safe-area-view';
+import MapView from 'react-native-maps';
+import {Marker} from 'react-native-maps';
+
+const SingleSelfGuided = ({navigation, tour}) => {
+  return (
+    <>
```
Comment (siddharthparmar7) [inline client/js/screens/SingleSelfGuided/SingleSelfGuided.js]: Please remove your Fragment wrapper. No need as the SafeAreaView is the only child it has.

File: client/js/screens/SingleSelfGuided/SingleSelfGuided.js [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+                </View>
+              </View>
+            </View>
+            <Text style={styles.infoText}>Audio</Text>
+            <View style={styles.inputFieldStyles}>
+              <Image
+                style={styles.eventTextFields}
+                source={require('../../assets/images/icFilterAudio.png')}
+              />
+              <Text style={{fontSize: 12}}>
```
Comment (siddharthparmar7) [inline client/js/screens/SingleSelfGuided/SingleSelfGuided.js]: Move your inline style to the styles.js file

File: client/js/screens/SingleSelfGuided/SingleSelfGuidedContainer.js [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+import {QUERY_SELFGUIDED_TOUR} from '../../apollo/queries';
+import Loader from '../../components/Loader';
+import {Text, TouchableOpacity, Image} from 'react-native';
+
+class SingleSelfGuidedContainer extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      mockId: 'ck435ktpmunrv0b00xbq37xme',
```
Comment (siddharthparmar7) [inline client/js/screens/SingleSelfGuided/SingleSelfGuidedContainer.js]: what is this id for? Can you use dynamic data?

File: client/js/screens/SingleSelfGuided/SingleSelfGuidedContainer.js [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+            }}>
+            <Image source={require('../../assets/images/icArrLeftWhite.png')} />
+          </TouchableOpacity>
+        );
+      },
+    };
+  };
+
+  render() {
+    console.log(this.props.navigation.state);
```
Comment (siddharthparmar7) [inline client/js/screens/SingleSelfGuided/SingleSelfGuidedContainer.js]: Please remove this console log.

File: client/js/screens/SingleSelfGuided/SingleSelfGuidedContainer.js [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+    };
+  }
+
+  static navigationOptions = ({navigation}) => {
+    return {
+      title: 'SingleSelfGuided',
+      headerLeft: () => {
+        return (
+          <TouchableOpacity
+            style={{marginLeft: 12}}
```
Comment (siddharthparmar7) [inline client/js/screens/SingleSelfGuided/SingleSelfGuidedContainer.js]: Please move inline styles to styles.js file

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
Please resolve merge conflicts and make requested changes. Thank you.

File: client/js/screens/SingleSelfGuided/SingleSelfGuided.js (hunk 1 above)
Comment (alefaissal) [reply client/js/screens/SingleSelfGuided/SingleSelfGuided.js]: Removed

File: client/js/screens/SingleSelfGuided/SingleSelfGuided.js (hunk 2 above)
Comment (alefaissal) [reply client/js/screens/SingleSelfGuided/SingleSelfGuided.js]: Moved

File: client/js/screens/SingleSelfGuided/SingleSelfGuidedContainer.js (hunk 3 above)
Comment (alefaissal) [reply client/js/screens/SingleSelfGuided/SingleSelfGuidedContainer.js]: Removed

File: client/js/screens/SingleSelfGuided/SingleSelfGuidedContainer.js (hunk 4 above)
Comment (alefaissal) [reply client/js/screens/SingleSelfGuided/SingleSelfGuidedContainer.js]: Removed

File: client/js/screens/SingleSelfGuided/SingleSelfGuidedContainer.js (hunk 5 above)
Comment (alefaissal) [reply client/js/screens/SingleSelfGuided/SingleSelfGuidedContainer.js]: Moved

**Review by siddharthparmar7 — APPROVED** [review]

**Pull request merged**